services/uploads/serverless.yml

```diff
@@ -163,16 +163,6 @@ resources:
         Bucket: !Ref AttachmentsBucket
         PolicyDocument:
           Statement:
-            - Action:
-                - "s3:GetObject"
-              Effect: "Deny"
-              Resource: !Sub ${AttachmentsBucket.Arn}/*
-              Principal: "*"
-              Condition:
-                StringNotEquals:
-                  s3:ExistingObjectTag/virusScanStatus:
-                    - "CLEAN"
```

Comment:
This deny statement is needed to prevent unscanned files from being downloaded.

Reply:
Added back policy and manually tagged templates while we figure out what's going on with virus scanner